site/app/templates/polls/PollForm.twig

```diff
@@ -202,6 +221,36 @@
         }
         submitErrorChecks();
     };
+    window.onload = function() {
+        toggleTimerInputs();
+    };
+    function toggleTimerInputs() {
+        var enableTimerCheckbox = document.getElementById("enable-timer");
+        var timerInputs = document.getElementById("timer-inputs").querySelectorAll(".poll-timer");
+
+        if (enableTimerCheckbox.checked) {
+            document.getElementById("timer-inputs").style.display = "block";
```

Comment:
You can use `.show()` instead of manipulating the style directly.

Reply:
Fixed, changed function to use jquery library.